ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
@avtar @gtirloni - I've had the container install to `/opt/gpii/node_modules/universal' - this allows the tests to be run and resolve successfully.

The issue I see now is that failure of `npm test` (at least in this context) does not send a non-0 exit code. This occurs whether `npm test` is run by the Ansible role (via `nodejs_app_commands`) or directly by the Dockerfile.

I'm not sure we should be working around this at the container build level if a test failure is exiting with a 0 error code. This seems like something that should be fixed at the test code level, as the tests are not useful in a CI process if they exit 0 on failure.

Example of this running the tests manually from within the container, with universal installed to `opt/universal`.

```
[root@3c5eb72483ab universal]# node tests/all-tests.js
15:16:50.740:  Registering module infusion from path /opt/universal/node_modules/infusion
15:16:50.748:  Registering module kettle from path /opt/universal/node_modules/kettle
15:16:50.835:  FATAL ERROR: Uncaught exception: Cannot find module 'universal'
Error: Cannot find module 'universal'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/opt/universal/tests/all-tests.js:24:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:929:3
[root@3c5eb72483ab universal]# echo $?
0
[root@3c5eb72483ab universal]#
```